src/TestSuite/IntegrationTestCase.php

```diff
@@ -945,7 +945,7 @@ public function assertSession($expected, $path, $message = '')
      */
     public function assertCookie($expected, $name, $message = '')
     {
-        if (empty($this->_response)) {
+        if (!$this->_response) {
```

Comment:
we look for wider coverage in tests. I would prefer the empty. This is a test after all and empty covers more edge cases.